applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -120,6 +120,12 @@ int UPwSmallStrainInterfaceElement<TDim,TNumNodes>::
     KRATOS_CATCH( "" );
 }
 
+template< unsigned int TDim, unsigned int TNumNodes >
```

Comment:
Two different integration methods are used in interface elements. Lobatto and Gauss integration methods. Lobatto is used for the calculation and stored in `mThisIntegrationMethod`. The Gaussian integration is used for the post-processing (GiD) and you can get it by calling `this->GetIntegrationMethod() `. I guess, now that you are overriding `GetIntegrationMethod()` and returning `mThisIntegrationMethod`, you will not be able to open the results in GiD. Please check if this is the case.